### PR TITLE
Update devstack.sh

### PR DESCRIPTION
**Pull request by lyfofvipin**

### CODE REVIEW

File: devstack.sh [hunk 1]
```diff
@@ -11,5 +12,4 @@ echo "ADMIN_PASSWORD=secret" >> local.conf
 echo "DATABASE_PASSWORD=$ADMIN_PASSWORD" >> local.conf
 echo "RABBIT_PASSWORD=$ADMIN_PASSWORD" >> local.conf
 echo "SERVICE_PASSWORD=$ADMIN_PASSWORD" >> local.conf
-sudo chmod 755 /opt/stack
```
Comment (son-vyas) [inline devstack.sh:14]: Vipin3699, If we look at line 5  we have switched user from root to stack. As a result, chmod will require sudo privileges.

File: devstack.sh [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #! /bin/sh
 useradd -s /bin/bash -d /opt/stack -m stack
+chmod 755 /opt/stack
 echo "stack ALL=(ALL) NOPASSWD: ALL" | tee /etc/sudoers.d/stack
 su - stack
 yum -y install git
```
Comment (son-vyas) [inline devstack.sh:6]: vipin3699, This step will require sudo privilege for yum to install git.

File: devstack.sh (hunk 1 above)
Comment (lyfofvipin) [reply devstack.sh:14]: that's why I change the line position so we don't even need to use sudo

File: devstack.sh (hunk 2 above)
Comment (lyfofvipin) [reply devstack.sh:6]: will change that asap

**Pull request closed without merging**